src/inverseRobot.js

```diff
@@ -7,7 +7,18 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const reverse = Object.entries(robot).map(([value, key]) => [key, value]);
+
+  const flatted = reverse.flat();
+  const uniqueParts = flatted
+    .filter((element, index, array) => array.indexOf(element)
+    === index);
```

Comment:
you can use Set for it
```suggestion
  const uniqueParts = new Set(...reverse);
```

Reply:
i didn't get how to use Set instead of my solution so i prefer to leave my option with filter 😅

Reply:
You can try to do it 😀 Set contains ONLY unique values